playbooks/vars/sample-variables-sap-hypervisor-node-preconfigure-rh_ocp_virt.yml

```diff
@@ -1,3 +1,8 @@
+sap_hypervisor_node_preconfigure_install_operators: true
```

Comment:
I see these variables are not documented, suggestions is to set defaults and document them

Reply:
It is defined here:

`sap_hypervisor_node_preconfvars/platform_defaults_redhat_ocp_virtp_virt.yml

```
# Should the operators be installed
sap_hypervisor_node_preconfigure_install_operators: True
```